clcache.py

```diff
@@ -74,6 +86,78 @@ def __str__(self):
         return repr(self.message)
 
 
+class ManifestsManager(object):
+    # Bump this counter whenever the current manifest file format changes.
+    # E.g. changing the file format from {'oldkey': ...} to {'newkey': ...} requires
+    # invalidation, such that a manifest that was stored using the old format is not
+    # interpreted using the new format. Instead the old file will not be touched
+    # again due to a new manifest hash and is cleaned away after some time.
+    HASH_INVALIDATION_COUNTER = 3
```

Comment:
A counter which is changed whenever some format changes is often just called a 'version number'. So how about this gets renamed to `MANIFEST_FORMAT_VERSION`?